gridcapa-core-cc-app/src/main/java/com/farao_community/farao/gridcapa_core_cc/app/domain/CoreCCTaskParameters.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.gridcapa_core_cc.app.domain;
+
+import com.farao_community.farao.gridcapa.task_manager.api.TaskParameterDto;
+import com.farao_community.farao.gridcapa_core_cc.api.exception.CoreCCInvalidDataException;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Daniel Thirion {@literal <daniel.thirion at rte-france.com>}
+ */
+public class CoreCCTaskParameters {
+    private static final Logger LOGGER = LoggerFactory.getLogger(CoreCCTaskParameters.class);
+    private static final String USE_DC_CGM_INPUT = "USE_DC_CGM_INPUT";
+    private static final String KEY_VALUE_FORMAT = "%n\t\"%s\": %s";
+    private boolean useDcCGmInput;
```

Comment:
Fix attribute's name: "CGm" (and adapt the name of corresponding getter function)

Reply:
The name of the getter is still not written correctly (at line 54)